tla/consistency/ExternalHistoryInvars.tla

```diff
@@ -205,7 +205,61 @@ CommittedRwLinearizableInv ==
     /\ AllCommittedObservedInv
     /\ AtMostOnceObservedInv
 
-\*Debugging invariants to check that specific states are reachable
+----
+
+\* TxIDs are of the form <<view,seqnum>>
+View(tx_id) == tx_id[1]
+SeqNum(tx_id) == tx_id[2]
+
+\* Ordering over txIDs
+TxIDStrictlyLessThan(x, y) ==
+    \/ View(x) < View(y)
+    \/ /\ View(x) = View(y)
+       /\ SeqNum(x) < SeqNum(y)
+
+\* CommittedRwResponse is a subsequence of history containing only the responses to committed rx transactions
+\* and sorted by tx_id (instead of by event ordering)
+CommittedRwResponses == 
```

Comment:
`TxIDOrderedCommittedRwResponses`, or `OrderedByTxIDCommittedRwResponses` would be more descriptive but perhaps too long.